qiskit/providers/aer/backends/aerbackend.py

```diff
@@ -100,21 +104,22 @@ def __init__(self,
         self._available_methods = [] if available_methods is None else available_methods
 
         # Set custom configured options from backend_options dictionary
-        self._options = {}
         if backend_options is not None:
             for key, val in backend_options.items():
-                self._set_option(key, val)
+                self.set_option(key, val)
 
     # pylint: disable=arguments-differ
+    @deprecate_arguments({'qobj': 'circuits'})
     def run(self,
-            qobj,
+            circuits,
             backend_options=None,  # DEPRECATED
             validate=False,
             **run_options):
```

Comment:
are the run_options documented somewhere? I think it should be documented here so the backend.run() user knows what kind of options are valid. Especially now that things like number of shots which used to be in qobj have to be passed here.

Reply:
Here are all the run options for mps:
The following options are unique for matrix_product_state:
`matrix_product_state_truncation_threshold`: double
Discard the smallest coefficients for which the sum of their squares is smaller than this threshold.   (Default: 1e-16).
`matrix_product_state_max_bond_dimension` – uint
Sets a limit on the number of Schmidt coefficients retained at the end of the svd algorithm. Coefficients beyond this limit will be discarded.  (Default: None, i.e., no limit on the bond dimension).
`mps_sample_measure_algorithm` – `mps_probabilities`/ `mps_apply_measure`
Choose which algorithm to use for `sample_measure`. `mps_probabilities` means all state probabilities are computed and measurements are based on them. It is more efficient for a large number of shots, small number of qubits and low entanglement. `mps_apply_measure` creates a copy of the mps structure and makes a measurement on it. It is more efficient for a small number of shots, high number of qubits, and low entanglement. If the user does not specify the algorithm, a heuristic algorithm is used to select between the two algorithms.      (Default: `mps_heuristic`).

The following options are applicable for other simulation methods as well as mps.
`Chop_threshold` – double
Cutoff threshold for precision of computations involving floating point numbers. (Default: 1E-8)
`mps_parallel_threshold` – uint
Threshold for enabling omp. (Default: 14)

`mps_omp_threads` – uint
Number of threads when using omp. (Default: 1)